lookout/style/format/visualizer/server.py

```diff
@@ -2,21 +2,22 @@
 from collections.abc import Mapping as Mapping_abc
 import logging
 from pathlib import Path
-from typing import Any, List, Mapping, MutableMapping
+from typing import Any, Dict, List, Mapping, MutableMapping, Union
 
 from bblfsh import BblfshClient
 from flask import abort, Flask, jsonify, request, Response
 from flask_cors import CORS
 from lookout.core.api.service_data_pb2 import File
 import numpy
+from scipy.sparse import csr_matrix
 
 from lookout.style.format.descriptions import describe_rules, describe_sample
 from lookout.style.format.feature_extractor import FeatureExtractor
 from lookout.style.format.model import FormatModel
 from lookout.style.format.postprocess import filter_uast_breaking_preds
 from lookout.style.format.virtual_node import VirtualNode
 
-logging.basicConfig(level="INFO")
+logging.basicConfig(level="DEBUG")
```

Comment:
default one should be INFO I think.

Reply:
This is a debugging tool, default one should be DEBUG